adding geocoords and timezones

kengie opened the pull request:
This is to run against the dispatch_all_geocoded table for that flight. I update the other ones too with timezones.

Commit:
adding goecoords and timezones

jokedurnez commented on IowaPD_dispatch.yaml:
```diff
@@ -469,6 +470,12 @@ entityDefinitions:
+      ol.locationcoordinates:
+        type: ol.locationcoordinates
+        transforms:
+        - !<transforms.ConcatTransform>
```
We also have now:
```
- !<transforms.GeographyPointTransform>:
    latTransforms:
        - !<transforms.ColumnTransform>:
          column: Latitude
    lonTransforms:
        - !<transforms.ColumnTransform>:
          column: Longitude
```
It would be more lines here, but guarantees an error is thrown with invalid input...

kengie replied:
sounds good, I changed it. Can you check that the spacing is right in the code?

kengie committed:
updating location coordinate code

I changed it. Can you double check that the spacing is right, not sure about that.
